src/qt/lookuptxdialog.cpp

```diff
@@ -0,0 +1,177 @@
+
+    // first let's check if we have a searchText, if not do nothing
+    if (searchText.empty()) return;
+
+    uint256 hash;
+    hash.SetHex(searchText);
+    Object txobj;
+    // make a request to new RPC populator function to populate a transaction object
+    int populateResult = populateRPCTransactionObject(hash, &txobj, "");
+    if (0<=populateResult)
+    {
+        std::string strTXText = write_string(Value(txobj), false) + "\n";
+        // clean up
+        string from = ",";
+        string to = ",\n    ";
+        size_t start_pos = 0;
+        while((start_pos = strTXText.find(from, start_pos)) != std::string::npos)
+        {
+            strTXText.replace(start_pos, from.length(), to);
+            start_pos += to.length(); // Handles case where 'to' is a substring of 'from'
+        }
+        from = ":";
+        to = "   :   ";
+        start_pos = 0;
+        while((start_pos = strTXText.find(from, start_pos)) != std::string::npos)
+        {
+            strTXText.replace(start_pos, from.length(), to);
+            start_pos += to.length(); // Handles case where 'to' is a substring of 'from'
+        }
+        from = "{";
+        to = "{\n    ";
+        start_pos = 0;
+        while((start_pos = strTXText.find(from, start_pos)) != std::string::npos)
+        {
+            strTXText.replace(start_pos, from.length(), to);
+            start_pos += to.length(); // Handles case where 'to' is a substring of 'from'
+        }
+        from = "}";
+        to = "\n}";
+        start_pos = 0;
+        while((start_pos = strTXText.find(from, start_pos)) != std::string::npos)
+        {
+            strTXText.replace(start_pos, from.length(), to);
+            start_pos += to.length(); // Handles case where 'to' is a substring of 'from'
+        }
+
+        QString txText = QString::fromStdString(strTXText);
+        QDialog *txDlg = new QDialog;
+        QLayout *dlgLayout = new QVBoxLayout;
+        dlgLayout->setSpacing(12);
+        dlgLayout->setMargin(12);
+        QTextEdit *dlgTextEdit = new QTextEdit;
+        dlgTextEdit->setText(txText);
+        dlgTextEdit->setStatusTip("Transaction Information");
+        dlgLayout->addWidget(dlgTextEdit);
+        txDlg->setWindowTitle("Transaction Information");
+        QPushButton *closeButton = new QPushButton(tr("&Close"));
+        closeButton->setDefault(true);
+        QDialogButtonBox *buttonBox = new QDialogButtonBox;
+        buttonBox->addButton(closeButton, QDialogButtonBox::AcceptRole);
+        dlgLayout->addWidget(buttonBox);
+        txDlg->setLayout(dlgLayout);
+        txDlg->resize(700, 360);
+        connect(buttonBox, SIGNAL(accepted()), txDlg, SLOT(accept()));
+        txDlg->setAttribute(Qt::WA_DeleteOnClose); //delete once it's closed
+        if (txDlg->exec() == QDialog::Accepted) { } else { } //do nothing but close
+    }
+    else
+    {
+        // show error message
+        string strText = "The transaction ID entered was not valid.";
```

Comment:
This is not always true. More accurate would probably something like "not a valid, confirmed Omni transaction".